src/tirith/core/core.py

```diff
@@ -82,7 +82,24 @@ def generate_evaluator_result(evaluator_obj, input_data, provider_module):
                 continue
 
             evaluation_result = evaluator_instance.evaluate(evaluator_input["value"], evaluator_data)
-            evaluation_result["meta"] = evaluator_input.get("meta")
+            meta = evaluator_input.get("meta")
```

Comment:
I would revert this change so that the `meta` key will always be included even though it's null -- for backward compatibility